hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/cluster/SparkExecuteClusteringCommitActionExecutor.java

```diff
@@ -118,7 +118,7 @@ public SparkExecuteClusteringCommitActionExecutor(HoodieEngineContext context,
       Schema readerSchema = HoodieAvroUtils.addMetadataFields(new Schema.Parser().parse(config.getSchema()));
       return ((ClusteringExecutionStrategy<T, JavaRDD<HoodieRecord<? extends HoodieRecordPayload>>, JavaRDD<HoodieKey>, JavaRDD<WriteStatus>>)
           ReflectionUtils.loadClass(config.getClusteringExecutionStrategyClass(), table, context, config))
-          .performClustering(inputRecords, clusteringGroup.getNumOutputFileGroups(), instantTime, strategyParams, readerSchema);
+          .performClustering(inputRecords, 0, instantTime, strategyParams, readerSchema);
```

Comment:
why is this relevant here?

Reply:
The build was failing and not able to find this because I probably hadn't built the avro classes, so these changes were just a workaround, they have been reverted now.